build/build.go

```diff
@@ -75,7 +75,19 @@ func Build(ctx context.Context, drivers []driver.Driver, opt map[string]Options,
 			FrontendAttrs: map[string]string{},
 		}
 
-		if len(opt.Exports) > 1 {
+		switch len(opt.Exports) {
+		case 1:
+			// valid
+		case 0:
+			if _, ok := d.(interface {
+				IsDefaultMobyDriver()
+			}); !ok {
+				return nil, errors.Errorf("loading to docker currently not implemented, specify dest file or -")
```

Comment:
This should only be shown if exporter type is docker and there is no dest set

Reply:
`docker` or `image` ?

Reply:
docker